pkg/cmd/resources.go

```diff
@@ -2,17 +2,45 @@ package cmd
 
 import (
 	"fmt"
+	"io/ioutil"
+	"net/http"
 
+	"github.com/pelletier/go-toml"
 	"github.com/spf13/cobra"
 
 	"github.com/stripe/stripe-cli/pkg/ansi"
 	"github.com/stripe/stripe-cli/pkg/validators"
 )
 
+const (
+	pluginManifestUrl = "https://stripe.jfrog.io/artifactory/stripe-cli-plugins-local/plugins.toml"
+)
+
 type resourcesCmd struct {
 	cmd *cobra.Command
 }
 
+// PluginList contains a list of plugins
+type PluginList struct {
+	Plugins []Plugin `toml:"Plugin"`
+}
+
+// Plugin contains the plugin properties
+type Plugin struct {
+	Shortname        string
+	Binary           string
+	Releases         []Release `toml:"Release"`
+	MagicCookieValue string
+}
+
+// Release is the type that holds release data for a specific build of a plugin
+type Release struct {
+	Arch    string
+	OS      string
+	Version string
+	Sum     string
+}
+
```

Comment:
~~Are we able to import from the plugins package in the `resources_test.go` file instead to make use of the same types defined in there? Then we can remove these. What do you think?~~

See my note about `getPluginCommands` instead 😃